### DISCUSSION TITLE
Collection and Diachronic Works Marker Questions

### DISCUSSION THREAD

**Discussion by tmqdeborah** [crystalyragui/MARC2RDA#467] [General]
Here are questions about markers for identifying Collection Works and Diachronic Works.

======================================================
Please do not start a new comment to reply to a question; use **REPLY** instead ======================================================

These markers are pattern matches that will (hopefully) reliably identify that a MARC record is describing one of the following types of works:

- Collection works with [a plan for selecting items that are to be gathered and kept together for some purpose](https://access.rdatoolkit.org/en-US_rdaregistry.info-termList-RDATerms-1210).
- Diachronic woks with a plan [to be embodied over time](https://access.rdatoolkit.org/en-US_rdaregistry.info-termList-RDATerms-1131).

**Comment by tmqdeborah** [comment]
### Question: If a MARC record is describing a collection of items that have been reproduced in digital format is that record describing a Collection work or an Aggregating work?

@GordonDunsire, last year I asked you (in an email dated September 18, 2023) whether a MARC record that describes a microform reproduction of a collection of items is describing a Collection work. Your reply was, "Bottom line is that a reproduction of a collection of items must be an aggregate collection of expressions that are exemplified by the items. Reproduction of an item or a manifestation always results in a new manifestation that embodies the expressions that are associated with the original item or manifestation."

I just want to confirm that this means that if a MARC record is coded as a collection (000/07='c') but carries indications that the carrier type of the manifestation is a microform or a computer disc (etc.) or online resource (e.g., in a300$a), then that record is describing an Aggregating work, not a Collection work.

**Reply by CECSpecialistI** [reply]
@tmqdeborah @GordonDunsire I think I follow this. But, will the logic catch collections of computer discs or microforms which are not reproductions of other collections but collections in themselves?

**Reply by tmqdeborah** [reply]
Good question. I can narrow the search to:
AND 000/07=c, NOT 533

That is, the logic will only catch records that are coded as 'collections' but are not also coded as 'reproductions'.

**Reply by GordonDunsire** [reply]
@tmqdeborah: My earlier response relates to reproductions (your original question). It does not relate to non-reproductions.

Luckily, the answer is the same. It is possible to create a microfilm strip be gluing (literally) together shorter strips, and to create a computer tape by gluing shorter tapes (primarly punched paper tapes). Both result in 'bound-with' manifestations, which RDA now defines as collection manifestations with associated collection works. Of course, this does not apply to microfiche or computer discs; those cases involve some form of copying or reproduction and are covered by the original question.

**Comment by tmqdeborah** [comment]
### Question: Do you agree that a 500 note "Kept up to date by supplements" indicates a description of a diachronic work?

Even if a record is coded as a monograph (000/07=m) and a single date is give as date of publication (26X$c), this note seems (to me) to be indicating a plan for content to be embodied over time. Do you agree or disagree?

**Reply by CECSpecialistI** [reply]
Oh my. I think this is going a step too far. This seems to be describing things as minor as errata.

**Reply by tmqdeborah** [reply]
A search on "Kept up to date by" in the LC online [catalog ](https://catalog.loc.gov/ ) finds over 7000 records that most commonly seems to contain, e.g., 
- Kept up to date by quarterly supplements. [would already have been pulled by an open ended publication date (260$c1988- )]
- Kept up to date by cumulative supplements. [would _not_ already have been pulled by an open ended publication date (260$c1990)]
- Kept up to date by pocket parts and revised volumes. (also has open ended publication date)
-  Kept up to date by supplements. (not an open ended publication date)

Adding "errata" to that phrase ("Kept up to date by" errata) only found 15 records and "errata" was in a separate 500 field.

**Reply by lake44me** [reply]
In MARCspeak, these sound like "integrating" resources and by nature those would be diachronic.  But, they are coded in the leader as monograph.  Strange animals where the updates may or may not be integrated into the original published volumes (if you call a pocket part an integration).  

I guess "Kept up to date by" in a note could be considered a marker of diachronicity, but the open dates are maybe a more reliable clue.

**Comment by tmqdeborah** [comment]
### Question: If a MARC record is describing a collection of items that have been duplicated for preservation purposes is that record describing a Collection work or an Aggregating work?

I **think** that the answer might be that these are all 'reproductions' and therefore are not collected works, despite being coded as such as 000/07=c.

If this is so, then it will be a bit of a challenge eliminating then from the 000/07=c pattern, because of the variety of ways that the reproduction characteristic is recorded in notes:
- 500$a = "duplication project" OR "duplicated from" OR "copies of" OR "Preservation master:"

For example:
- original field recordings on '1 track' sound tape reels that have been 'duplicated' on '2 track' sound tape reels
245 00$aKenneth S. Goldstein collection of North and South Carolina recordings$h[sound recording].
300   $a3 sound tape reels (5 hrs. 30 min.) :$banalog, 7 1/2 ips, 2 track ;
500   $aLibrary of Congress duplication project, 1972. Original field recordings on 19 sound tape reels: analog, 7 1/2 and 15 ips, 1 track; 7 in.
520   $aCollection of field recordings of folksongs, ballads, and instrumental music, including fiddle, banjo, guitar, harmonica, and mandolin, recorded in Asheville and Hot Springs, North Carolina, and in South Carolina by Kenneth S. Goldstein during July 1956 and July 19- 23, 1957.  Nathaniel Thackston was recorded singing hymns and spirituals in Greenville, South Carolina in July 1956

- duplicated sound recordings
245 00$aHerbert Halpert New York and Pennsylvania recordings$h[sound recording].
300   $a5 sound discs :$banalog ;
500   $aDuplicated from originals disc recordings made by Herbert Halpert in the Columbia University Folk-Song Collection.
520   $aCollection of field recordings of girls' songs, rhymes, and chants recorded at Camp Winona, Pennsylvania; plus ballads, folk songs, and a hymn, performed by Mort Montonyea, William H. Dickinson, and John Harris at various locations in New York City and New York state from 1937 to 1939 by Herbert Halpert.

- 'copied' sound recordings
245 00$aBentley Historical Library collection$h[sound recording].
300   $a113 sound discs :$banalog ;
300   $a52 sound tape reels :$banalog ;
300   $a27 sound tape reels :$banalog ;
300   $a47 sound cassettes :$banalog.
300   $a1
500   $aCollection includes disc and tape copies of selected recordings from the Alan Lomax Collection of Michigan and Wisconsin Recordings (AFC 1939/007) recorded by Alan Lomax, August 10-November 1, 1938 for the Archive of Folk Song, Library of Congress.

- duplicated videos from a Preservation master
245 04$aThe Flatt and Scruggs show$nNo. 383,$nNo. 384$h[videorecording] /$cWSM-TV.
300   $a2 videoreels of 2 (ca. 60 min.) :$bsd., col. ;
500   $aVariant title from slate.
500   $aTwo half-hour television programs, edited 1968-04-10, broadcast on WSM-TV, Channel 4, [Nashville, Tennessee]
500   $aPreservation master:  2 videocassettes of 2 (Betacam SP), created in 2003, from 2 video reels, 2-inch, Hi-Band.

What about the original recordings? I think this **is** a collection of sound tape reels:
245 00$aAlan Jabbour recordings of Jehile Kirkhuff and John Vesey$h[sound recording].
300   $a2 sound tape reels :$banalog, 7 1/2 ips, full track ;
300   $a7 sound tape reels :$banalog, 7 1/2 ips, full track ;
300   $a1
520   $aField recordings of fiddle tunes and interviews conducted by Alan Jabbour with two fiddlers, John Vesey, born in Sligo County, Ireland, who immigrated to the Philadelphia, Pennsylvania area in 1948, and who performed in the style of Sligo fiddlers; and Jehile Kirkhuff, who was born and raised near Rush, Susquehanna County, Pennsylvania, who learned fiddle tunes from other musicians, radio, phonograph records, and publications.
